client/me/help/help-contact/index.jsx

```diff
@@ -247,6 +255,10 @@ module.exports = React.createClass( {
 
 		if ( olark.details.isConversing && olark.isOperatorAvailable ) {
 			return <OlarkChatbox />;
+		} else {
+			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
+				form_type: showKayakoVariation ? 'kayako' : 'forum'
+			} );
```

Comment:
I'm not sure but I think we might end up with a bunch of these events because of multiple re-renders that aren't related to the form switching. I'd consider moving this to the `onOperatorsAway` method and wrapping it with the proper conditionals.